css-three-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  color: white;
+  font-family: sans-serif, 'Courier New', Courier;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+  justify-content: center;
+  align-items: center;
+}
+
+.row {
+  display: flex;
+  width: 90%;
+  justify-content: center;
+}
+
+.column-full {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  padding-top: 60px;
+}
+
+.column-full h1 {
+  font-weight: lighter;
+}
+
+.column-third {
+  width: 33%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 50px;
+  padding: 20px;
```

Comment:
Same deal here, use separate classes from your `column` class to do this work. You want your column classes to remain reusable, and this makes them too specific to a single situation.